rfc/rfc004-verifiable-transactional-graph.md

```diff
@@ -65,6 +65,10 @@ In addition to the registered header parameters, the following headers MUST be p
   * When it's a root transaction the field SHALL NOT have any entries.
   * When creating a transaction it MUST only contain transactions that the local node has successfully processed, to avoid publishing unprocessable transactions.
 
+The following protected headers MAY be present:
+
+* **to**: MUST contain the encrypted address of the recipient \(used for private transactions, see section 3.8\).
```

Comment:
It might, but I'm not sure. Since this spec controls all implementation, every custom header is critical, by spec. Unless we spec a header as "you may use this header, but you do not need to process it". But that header should probably be left out anyways.

I'll add it to `crit`.